core/src/test/java/com/linecorp/armeria/server/ConnectionLimitingHandlerTest.java

```diff
@@ -44,13 +50,18 @@ void testExceedMaxNumConnections() {
 
     @Test
     void testMaxNumConnectionsRange() {
-        final ConnectionLimitingHandler handler = new ConnectionLimitingHandler(Integer.MAX_VALUE);
+        final Server server = Server.builder()
```

Comment:
Question) Couldn't we just create a `ServerMetrics` instance and pass it instead of creating a `Server`?
Ditto for the other test case above.

e.g.
```java
final ServerMetrics serverMetrics = new ServerMetrics();
val handler = new ConnectionLimitingHandler(Integer.MAX_VALUE, serverMetrics);
...
```

Reply:
Yes, we can create ServerMetrics!!
i think creating ServerMetrics is a better way for the test code!! 
i will apply the review thank you!! :)